enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -0,0 +1,327 @@
+  <!-- TODO: update non-goals based on the answer to number 3  -->
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+A community version of WMCO 8 or 9 can be released with incremental additions to Windows proxy support
+functionality, giving users an opportunity to get an early preview of the feature using OKD/OCP 4.13 or 4.14.
+It will also allow us to collect feedback to troubleshoot common pain points and learn if there are any shortcomings.
+
+#### Tech Preview -> GA
+
+The feature associated with this enhacement is targeted to land in the offical Red Hat operator version of WMCO 9.0.0
+within OpenShift 4.14 timeframe. The normal WMCO release process will be followed as the functionality described in this
+enhancement is integrated into the product.
+
+An Openshift docs update announcing Windows cluster-wide proxy support will be required as part of GA. The new docs
+should list any Windows-specific info, but linking to existing docs should be enough for overarching proxy/PKI details.
+
+#### Removing a deprecated feature
+
+N/A, as this is a new feature that does not supercede an existing one
+
+### Upgrade / Downgrade Strategy
+
+The relevant upgrade path is from WMCO 8.y.z in OCP 4.13 to WMCO 9.y.z in OCP 4.14. There will be no changes to the
+current WMCO upgrade strategy. Once customers are on WMCO 9.0.0:
+* if they have an existing cluster-wide proxy configured, their Windows nodes will begin utilizing it for egress traffic
+* otherwise, they can configure one and the Windows nodes will be automatically updated to use it by the operator
+
+When deconfiguring Windows instances, proxy settings will be cleared from the node. This involves undoing some node
+config steps i.e. unsetting proxy variables and deleting additional certificates from the machine's local trust store.
+This scenario will occur when upgrading both BYOH and Machine-backed Windows nodes.
+
+In both relevant OCP versions, OLM consumes global proxy settings and injects them into managed operators like WMCO.
+
+Downgrades are generally [not supported by OLM](https://github.com/operator-framework/operator-lifecycle-manager/issues/1177),
+which manages WMCO. In case of breaking changes, please see the
+[WMCO Upgrades](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator-upgrades.md#risks-and-mitigations)
+enhancement document for guidance.
+
+### Version Skew Strategy
+
+N/A. There will be no version skew since this work is all within 1 product subcomponent (WMCO). The 8.y.z version of the
+official Red Hat operator will not have cluster-wide egress proxy support for Windows enabled. Then, when customers move
+to WMCO 9.0.0, the proxy support will be available.
+
+## Implementation History
+
+The implementation history can be tracked by following the associated work items in Jira and source code improvements in
+the WMCO Github repo.
+
+Planned GA of the feature: WMCO v9.0.0 in OCP 4.14
```

Comment:
This can be removed given from here as it is called out in `Graduation criteria`.